Mathlib/Data/List/Basic.lean

```diff
@@ -599,7 +599,7 @@ theorem getElem_map_rev (f : α → β) {l} {n : Nat} {h : n < l.length} :
 
 theorem get_length_sub_one {l : List α} (h : l.length - 1 < l.length) :
     l.get ⟨l.length - 1, h⟩ = l.getLast (by rintro rfl; exact Nat.lt_irrefl 0 h) :=
-  (getLast_eq_getElem l _).symm
+  (getLast_eq_getElem _).symm
```

Comment:
`l` should stay explicit in `getLast_eq_getElem` since it's a rewriting lemma and `l` appears only in the `l ≠ []` hypothesis which itself will usually be found by unification with the LHS